style.css

```diff
@@ -264,16 +270,230 @@ body {
   padding-bottom: 15px;
   padding-top: 15px;
   border-radius: 10px;
-  margin-right: 20px;
+  gap: 10px;
   display: flex;
 }
 
-.m-l {
-  margin-left: 20px;
-}
-
 .language {
   display: flex;
   justify-content: space-between;
   padding-right: 60px;
 }
+
+@media only screen and (min-width: 768px) {
+  .section-a-container {
+    background-image: url("pimages/header-background-mobile.png");
+    background-position: center;
+    background-size: 100% 90%;
+    background-repeat: no-repeat;
+    display: flex;
+    padding: 0 14em;
+    justify-content: center;
+    align-items: start;
+    flex-direction: column;
+    text-align: start;
+  }
+
+  .navbar-content img {
+    display: none;
+  }
+
+  .navbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  .nav_link {
+    display: block;
+  }
+
+  .btn {
+    margin-right: -9px;
+  }
+
+  .nav_link ul {
+    display: flex;
+    list-style: none;
+    margin-top: 50px;
+    margin-right: 50px;
+  }
+
+  .nav_link ul a {
+    text-decoration: none;
+    color: black;
+    padding: 10px;
+    font-size: 15px;
+  }
+
+  .desktop {
+    display: grid;
+    grid-template-columns: 1fr;
+    align-items: center;
+    justify-content: center;
+    background-color: #f7f7f9;
+    padding-bottom: 5%;
+  }
```

Comment:
Kindly update your works section to match the figma design better🙏

Your design:
<img width="512" alt="image" src="https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/37116322/5aeda4af-aad0-4968-9d18-6273dc398f30">

Figma design:
<img width="492" alt="image" src="https://github.com/Ejikeokegbe/Mobile-First-portifolio/assets/37116322/8fb7485c-17ac-4c36-8fbe-813309c56feb">

A few things to note:
- Please reduce the left and right margins on each card
- Please reduce the left and right padding on each card
- Please increase the size of the image to match the figma design
- Please reduce the height of the text content so it doesn't span the full height of the card and matches the design